app/src/main/java/com/sopt/now/presentation/main/MainActivity.kt

```diff
@@ -3,42 +3,52 @@ package com.sopt.now.presentation.main
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
+import androidx.activity.viewModels
 import androidx.fragment.app.Fragment
 import com.sopt.now.R
 import com.sopt.now.databinding.ActivityMainBinding
 import com.sopt.now.model.User
 import com.sopt.now.presentation.auth.signup.SignUpActivity.Companion.USER_KEY
+import com.sopt.now.presentation.main.home.HomeFragment
+import com.sopt.now.presentation.main.my.MyFragment
+import com.sopt.now.presentation.main.search.SearchFragment
 import com.sopt.now.util.base.BaseActivity
 import com.sopt.now.util.ext.getParcelable
 
 class MainActivity : BaseActivity<ActivityMainBinding>({ ActivityMainBinding.inflate(it) }) {
```

Comment:
```suggestion
class MainActivity : BaseActivity<ActivityMainBinding>(ActivityMainBinding::inflate) {
```
제가 예전에 이 유틸 클래스 사용했을때에는 베이스 클래스에 메서드 레퍼런스로 넘기려고 람다로 넣었습니다 ㅎ

Reply:
아....!!
이거 자주 써야지 해놓고 여기에 적용할 생각은 못햇네요!!
감사합니다 ㅎㅎ